ci/before_install.sh

```diff
@@ -7,10 +7,6 @@ if [[ "${CI_TARGET}" == lint ]]; then
   exit
 fi
 
-if [[ "${TRAVIS_OS_NAME}" == osx ]]; then
-  >/dev/null brew update
```

Comment:
`>/dev/null` avoided the log noise. Is there a technical reason we need to use travis.yml instead?

Reply:
Maybe an issue could be filed against Travis to do the same?  Using a declarative syntax for what we want rather than having to maintain the code is useful.

Reply:
@jamessan I'll send them a patch and create a new PR should they merge it.

Reply:
For reference: removing the update in https://github.com/neovim/neovim/pull/10779 for performance reasons (takes ~3mins).

@mhinz 
Have you ever done a PR for Travis about/for this btw (just wondered)?